client_code/utils/__init__.py

```diff
@@ -13,36 +14,60 @@
 def __dir__():
     return [
         "auto_refreshing",
-        "wait_for_writeback",
-        "timed",
-        "logging",
         "BindingRefreshDict",
         "correct_canvas_resolution",
+        "import_module",
+        "timed",
+        "wait_for_writeback",
     ]
 
 
-@lru_cache(maxsize=None)
-def __getattr__(name):
-    # todo use dynamic imports but __import__ is not yet supported in skulpt
-    if name == "auto_refreshing":
-        from ._auto_refreshing import auto_refreshing
+def import_module(name, package=None):
+    """Import a module.
+
+    The 'package' argument is required when performing a relative import. It
+    specifies the package to use as the anchor point from which to resolve the
+    relative import to an absolute import.
+    """
+    level = 0
+    if name.startswith("."):
+        if not package:
+            msg = (
+                "the 'package' argument is required to perform a relative "
+                "import for {!r}"
+            )
+            raise TypeError(msg.format(name))
+        for character in name:
+            if character != ".":
+                break
+            level += 1
```

Comment:
could this just be:
``` python
level = name.count(".")
```

Reply:
It's just the first two dots in  `"..foo.bar"`

This part is identical to the implementation for `importlib.import_module`

Reply:
Gotcha. I misread it!